optimizationmodule/Function.go

```diff
@@ -0,0 +1,53 @@
+package optimizationmodule
```

Comment:
The real location should be "modules\prebid\optimization" with the package name "optimization".